docs/kubernetes/AWSIntegration.md

```diff
@@ -0,0 +1,140 @@
+# Integration with an AWS Kubernetes cluster
+This guide will walk you through how to use Calico Networking with an existing AWS Kubernetes cluster.
+
+## Requirements
+* A working Kubernetes Deployment on AWS with a Kubernetes master and one or more nodes (minions)
+    - We recommend the [Kubernetes instructions for AWS](https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/getting-started-guides/aws.md). This guide was created with the `kube-up` script in mind.
+* SSH access to your Kubernetes master and nodes
+    - Unless otherwise specified, the `kube-up` script will create a `kube_aws_rsa` private key in the `~/.ssh` folder which you can use to access your AWS Instances.
+    - SSH in with the following command `ssh -i </path/to/key> ubuntu@<PUBLIC_IP>`
+
+## Preparing your master services
+On your master, you will need to setup an etcd directory specifically for Calico. To do so, you will need to download our etcd manifest
+```
+wget https://raw.githubusercontent.com/projectcalico/calico-kubernetes-coreos-demo/master/manifests/calico-etcd.manifest
+```
+Replace all instances of `<PRIVATE_IPV4>` with your master's IP or hostname. Then, place the manifest file in the `/etc/kubernetes/manifests/` directory. 
+```
+sudo mv calico-etcd.manifest /etc/kubernetes/manifests/
+```
+After a short delay, the kubelet on your master will automatically create a container for the new etcd which can be accessed on port 6666 of your master.
+
+## Installing Calico
+On each of your AWS Instances, perform the following steps.
+* Download and install the `calicoctl` binary
+```
+wget https://github.com/projectcalico/calico-docker/releases/download/v0.9.0/calicoctl
+chmod +x calicoctl
+sudo mv calicoctl /usr/bin/
+```
+
+* Running Calico Node
+Use `calicoctl` to spin up the `calico-node` container.
+```
+sudo ETCD_AUTHORITY=<MASTER_PRIVATE_IPV4>:6666 calicoctl node --kubernetes --kube-plugin-version=v0.3.0
+```
+
+> Note: The `--kubernetes` flag will install the [networking plugin](https://github.com/projectcalico/calico-kubernetes) for you. You can also perform this operation manually with the following command.
+> ```
+> wget https://github.com/projectcalico/calico-kubernetes/releases/download/v0.3.0/calico_kubernetes
+> chmod +x calico_kubernetes
+> sudo mkdir -p /usr/libexec/kubernetes/kubelet-plugins/net/exec/calico/
+> sudo mv calico_kubernetes /usr/libexec/kubernetes/kubelet-plugins/net/exec/calico/calico
+> ```
+
+Next, set up an IP pool with IP-in-IP enabled. This is a [necessary step](https://github.com/projectcalico/calico-docker/blob/master/docs/FAQ.md#can-i-run-calico-in-a-public-cloud-environment) in any public cloud environment.
+> Note: You only need to call `pool add` once per cluster.
```

Comment:
You can probably have them run this on the master so that you don't need to include the `Note:` here.